Packs/AutofocusFeed/Integrations/AutofocusFeed/AutofocusFeed.py

```diff
@@ -0,0 +1,209 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# IMPORTS
+import re
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+SOURCE_NAME = "AutoFocusFeed"
+
+
+class Client(BaseClient):
+    """Client for AutoFocus Feed - gets indicator lists from the Custom and Daily threat feeds
+
+    Attributes:
+        api_key(str): The API key for AutoFocus.
+        insecure(bool): Use SSH on http request.
+        proxy(str): Use system proxy.
+        indicator_feeds(List): A list of indicator feed types to bring from AutoFocus.
+        output_feed_id(str): The ID given to the custom feed.
+        output_feed_name(str): The name given to the custom feed.
+    """
+    def __init__(self, api_key, insecure, proxy, indicator_feeds, output_feed_id="", output_feed_name=""):
+        self.url_feed_base_url = "https://autofocus.paloaltonetworks.com/api/v1.0/IOCFeed/"
+        self.daily_feed_base_url = "https://autofocus.paloaltonetworks.com/api/v1.0/output/threatFeedResult"
+        self.headers = {
+            "apiKey": api_key,
+            'Content-Type': "application/json"
+        }
+        self.indicator_feeds = indicator_feeds
+        self.url_feed_suffix = output_feed_id + "/" + output_feed_name
+        self.verify = not insecure
+        if proxy:
+            handle_proxy()
+
+    def http_request(self, feed_type) -> list:
+        """The HTTP request for the feed.
+
+        Args:
+            feed_type(str): The feed type (Daily or Custom feed).
+
+        Returns:
+            list. A list of indicators fetched from the feed.
+        """
+        if feed_type == "Daily Threat Feed":
+            url = self.daily_feed_base_url
+
+        else:
+            url = self.url_feed_base_url + self.url_feed_suffix
+
+        res = requests.request(
+            method="GET",
+            url=url,
+            verify=self.verify,
+            headers=self.headers
+        )
+        res.raise_for_status()
+        return res.text.split('\n')
+
+    def find_indicator_type(self, indicator):
+        """Infer the type of the indicator.
+
+        Args:
+            indicator(str): The indicator whose type we want to check.
+
+        Returns:
+            str. The type of the indicator.
+        """
+        if re.match(ipv4cidrRegex, indicator):
+            return FeedIndicatorType.CIDR
+
+        if re.match(ipv6cidrRegex, indicator):
+            return FeedIndicatorType.IPv6CIDR
+
+        if re.match(ipv4Regex, indicator):
+            return FeedIndicatorType.IP
+
+        if re.match(ipv6Regex, indicator):
+            return FeedIndicatorType.IPv6
+
+        elif re.match(sha256Regex, indicator):
+            return FeedIndicatorType.File
+
+        # in AutoFocus, URLs include '/' character while domains do not.
+        elif '/' in indicator:
```

Comment:
we have url regex, don't we?

Reply:
it doesnt work in this case as it seems domains and urls would both be identified in the same way.
after talking to @Arsenikr we figured this is the best solution.

Also we asked AutoFocus to tell us how they differentiate domains and urls and I'll update accordingly.

Reply:
small update - we got regexes from AutoFocus... they are quite complex and we got two followup questions we will ask:

1) Can we use them? - AutoFocus is not open source and it seems at least some of the regexes we got were modified by them. So we need to make sure if we can put them in the code before we just use them.

2) From the regexes, it seems the identification of Domains and URLs is almost identical in nature. So we need to understand if AutoFocus first check if and indicator is a Domain and if not then checks for URL or the other way around...

I am trying to figure out if there is a distinguishable difference between the two that can be implemented without the use of regexes... but I wouldn't hold a candle to that...